kodilla-hibernate/src/main/java/com/kodilla/hibernate/manytomany/facade/CompanySearchingException.java

```diff
@@ -0,0 +1,12 @@
+package com.kodilla.hibernate.manytomany.facade;
+
+public class CompanySearchingException extends Exception {
+
+    public static String ERR_EMPLOYEE_NOT_FOUND = "A string was provided with characters that are not available in the database for Employee ";
```

Comment:
Sugeruję nieco prostszy komunikat, np. "An employee with name containg provided characters not found".